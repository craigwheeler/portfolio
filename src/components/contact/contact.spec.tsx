import * as React from 'react';
import { render } from '@testing-library/react';
import ContactPage from '../contact';

describe('Contact', () => {
  it('should render contact page', () => {
    const elem = render(<ContactPage />).container;
    expect(elem).toMatchSnapshot();
  });
});
