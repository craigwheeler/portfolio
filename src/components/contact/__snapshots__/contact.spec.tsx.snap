// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render contact page 1`] = `
<div>
  <div
    class="sc-bZQynM kMEhVS"
  >
    <form
      class="sc-bdVaJa NFzwl"
      data-netlify="true"
      method="POST"
      name="portfolio-contact"
    >
      <div
        class="sc-bwzfXH ldcSgI"
      >
        <input
          name="form-name"
          type="hidden"
          value="portfolio-contact"
        />
        <input
          name="bot-field"
          type="hidden"
        />
        <div
          class="sc-htpNat bAXlYl"
        >
          Send Me A Message
        </div>
        <input
          class="sc-bxivhb kUkIgm"
          name="name"
          placeholder="Your Name"
          type="text"
        />
        <input
          class="sc-bxivhb kUkIgm"
          name="email"
          placeholder="Your Email Address"
          type="email"
        />
        <textarea
          class="sc-ifAKCX idTGTX"
          name="message"
          placeholder="Message"
        />
        <button
          class="sc-EHOje gQqEJd"
          type="submit"
        >
          Submit Message
        </button>
      </div>
    </form>
  </div>
</div>
`;
